swagger.yaml

```diff
@@ -56,7 +56,7 @@ paths:
       description: |
         Creates new Metaform
 
-        Method is allowed for system administrators and users with metaform administrators
+        Method is allowed for system administrators and metaform administrators
```

Comment:
This should be allowed only to system administators, because user cannot be metaform administrator before the form is actually created